integration_tests/tests/tests/test_main.py

```diff
@@ -66,6 +75,7 @@ def test_key_decorator_applied_by_exact_match_only():
     assert response.status_code == 200
     sdl = response.json()['data']['_service']['sdl']
     assert 'type FileNode  @key(fields: "id")' in sdl
+    assert 'type MultipleKeys  @key(fields: "id") @key(fields: "name")' in sdl
```

Comment:
I'd prefer to move this check in another test, as this test named: `test_key_decorator_applied_by_exact_match_only`